common/logger/init.go

```diff
@@ -0,0 +1,94 @@
+package logger
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"os"
+	"path/filepath"
+)
+
+const (
+	DefaultTimeFormat           = "15:04:05.000"
+	DefaultCallerSkipFrameCount = 4
+
+	UseColor  = true
+	UseCaller = false // if you want to expose line of code of caller
+	LogFile   = ""    // if you want to store log to file
```

Comment:
This isn't a very helpful constant. A user shouldn't be expected to edit this source file to get their logs written to a file. 

I don't think this option is really necessary. If they want the logs in a specific file, they can just pipe it from the command line.

But if you still want to leave this in, I suggest making it configurable with a command line option (e.g. --log_output=whatever.txt). 

In that case, change the "SetDebugMode" function to something like "ReadLogFlags" so both the debug mode and log output file can be read. Also, combine the registering of the two flags into a the single "AddLogFlag(s)" that I suggested above.

Reply:
The purpose of these constant is for developers not for user.
But i agree with you, they can just pipe it from the command line.
So i've removed it.